grpc-protobuf/logging.proto

```diff
@@ -11,6 +12,9 @@ service DoguLogMessages {
   // QueryForDogu queries logs for a dogu. This api-call is only available in a MultiNode CES with k8s-ces-control.
   // The logs are queried from k8s-loki and therefore it is only possible to query logs from period of max 30 days.
   rpc QueryForDogu(DoguLogMessageQueryRequest) returns (stream DoguLogMessage);
+
+  // SetLogLevel sets the log level for a specific dogu.
+  rpc SetLogLevel(LogLevelRequest) returns (google.protobuf.Empty);
```

Comment:
I was just informed that `SetLogLevel` not only sets a value but also restarts a dogu. This breaks the principle of least surprise given the doc comment and the method name.

I highly suggest to change both to reflect the possible downtime associated with a call. 

Maybe there would be another change be necessary, namely: If the default log level is in place (that is: no value was set) and the administrator sets the loglevel to a value equal to the loglevel. Under this circumstance the dogu's downtime could be avoided.

A rough idea of how the nature of dogu restart after a loglevel change could be reflected in the protobuf:

```suggestion
  // ApplyLogLevelWithRestart sets the log level for a specific dogu and restarts the dogu if the log level was changed.
  rpc ApplyLogLevelWithRestart(LogLevelRequest) returns (google.protobuf.Empty);
```

Reply:
Many thanks for the suggestion. I have implemented it.